easybuild/easyconfigs/x/XGrafix/XGrafix-2.41-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'XGrafix'
+version = '2.41'
+
+homepage = 'https://ptsg.egr.msu.edu/'
+description = """A graphics library and controller for physics simulation programs.
+3-d surface plots, scatter plots, 2-d line plots."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
```

Comment:
`XGrafix` can be built with MPI support. I would consider bumping to `gompi` and adding `--with-CC=$MPICC --with-CXX=$MPICXX` to the `configopts`

Reply:
I tried with `--enable-MPI --with-CC=mpicc --with-CXX=mpicxx` and I get:
```
configure: WARNING: unrecognized options: --enable-MPI
```
So no, that is not recognised. Besides, that is a X application. Whereas I could see the use of threads, I am not sure how MPI would work here.

Reply:
fair enough